helpers/typeConvert.js

```diff
@@ -32,34 +40,41 @@ const fromType = (propType, additionalProperties, items, shouldBox) => {
     case "boolean":
       return shouldBox ? "Boolean" : "boolean";
     case "string":
-      return "String";
+      return required_object_prefix + "String";
     case "array":
       return `List<${typeConvert(items, true)}>`;
     // inline object definition
     case "object":
       if (additionalProperties) {
         return `HashMap<String,${typeConvert(additionalProperties, true)}>`;
       } else {
-        return "Object";
+        return required_object_prefix + "Object";
       }
     default:
       return shouldBox ? "Void" : "void";
   }
 };
 
-const typeConvert = (prop, shouldBox = false) => {
+const typeConvert = (prop, shouldBox = false, inFnSignature = false) => {
   if (prop === null) return shouldBox ? "Void" : "void";
 
-  if (prop === undefined) return "object";
+  if (prop === undefined) return "Object";
 
   // resolve references
   if (prop.$ref) {
-    return toSafeName(prop.$ref.split("/").pop());
+    let required_object_prefix = !shouldBox && inFnSignature ? "@NonNull " : "";
```

Comment:
```suggestion
    const notNullPrefix = !shouldBox && inFnSignature ? "@NonNull " : "";
```

Reply:
I prefer "not null prefix" as a name because it makes more sense to me in the context of where it is used. This is subjective though, so totally up to you.